coretex/cli/modules/docker.py

```diff
@@ -0,0 +1,113 @@
+from typing import Dict, Any
+
+import json
+
+from ...utils import command, CommandException
+
+
+def isDockerAvailable() -> None:
+    try:
+        # Run the command to check if Docker exists and is available
+        command(["docker", "ps"], ignoreStdout = True, ignoreStderr = True)
+    except CommandException:
+        raise RuntimeError("Docker not available. Please check that it is properly installed and running on your system.")
+
+
+def networkExists(name: str) -> bool:
+    # This function inspects the specified Docker network using the
+    # 'docker network inspect' command. If the command exits with a return code
+    # of 0, indicating success, the function returns True, meaning the network exists.
+    # If the command exits with a non-zero return code, indicating failure,
+    # the function returns False, meaning the network doesn't exist.
+    try:
+        command(["docker", "network", "inspect", name], ignoreStdout = True, ignoreStderr = True)
+        return True
+    except:
+        return False
+
+
+def containerExists(name: str) -> bool:
+    try:
+        _, output, _ = command(["docker", "ps", "--format", "{{.Names}}"], ignoreStderr = True, ignoreStdout = True)
+        if name in output:
+            return True
+        return False
```

Comment:
Didn't we already discuss what is wrong with writing `if` statement in this case?